boot/boot_robustness_test.go

```diff
@@ -253,6 +266,13 @@ func (s *bootenv20Suite) TestHappySetNextBoot20KernelUpgradeUnexpectedReboots(c
 		expBlKernel       snap.PlaceInfo
 		comment           string
 	}{
+		{
+			"",                        // don't do any reboots for the happy path
+			s.kern2,                   // we should boot the new kernel
+			[]snap.PlaceInfo{s.kern2}, // expected kernel is new one
+			s.kern1,                   // the expected kernel is the old one
```

Comment:
the comments here need a tweak, because read at face value they are a bit confusing...  the are two sentences about expected kernels where the two kernels are different.  Maybe expected kernel before MarkSucceful on reboot and after ?

Reply:
okay I changed this now, is this better?